test/consts/const.go

```diff
@@ -18,5 +18,4 @@ const (
 	// MetalLBPeerCRDName contains the name of MetallB BGP Peer CRD
 	MetalLBPeerCRDName = "bgppeers.metallb.io"
 	// MetalLBConfigMapName contains created configmap
-	MetalLBConfigMapName = "config"
```

Comment:
I think it would be cleaner to have apply imported here - MetalLBConfigMapName = apply.MetalLBConfigMap and revert the changes to the tests. It would be clearer having all of the consts for the tests in a single place, worth the redundancy imo.

Reply:
I thought that way originally but wouldn't be cleaner to just have one constant for the entire code and test ?

Reply:
feels right to not mix tests and code too much and would look clearer having all tests vars in a single place I guess.